package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "pq-address",
+  "version": "1.0.0",
```

Comment:
This is version 1.0.0, but our Rust one is 0.1.0. Maybe we should just stick with 0.1.0